### DISCUSSION TITLE
How to open lsp_finder item by ENTER instead of 'o'

### DISCUSSION THREAD

**Discussion by haphamdev** [nvimdev/lspsaga.nvim#497] [Q&A] [answered]
By default, I can jump to the item in lsp_finder by pressing `o`. I would like to change it to `Enter` (`<CR>`). How to do it?
<img width="645" alt="image" src="https://user-images.githubusercontent.com/6322508/188262345-3428b014-68e3-44e1-a632-c8dc7b060010.png">

I tried
```
    use({
        "glepnir/lspsaga.nvim",
        branch = "main",
        config = function()
            local saga = require("lspsaga")

            saga.init_lsp_saga({
                -- your configuration
                saga_winblend = 30,
                finder_action_keys = {
                    open = "<CR>",
                    vsplit = "s",
                    split = "i",
                    tabe = "t",
                    quit = "q",
                    scroll_down = "<C-f>",
                    scroll_up = "<C-b>", -- quit can be a table
                },
            })
        end,
    })
```
But it does not work.
Config for `saga_winblend` does not work either.

Is there anything wrong with my config?

**Comment by glepnir** [comment, marked as answer]
works for me in test. scroll_down and scroll_up no longer support in 0.2.1 please check the example config.  you use the packer when you edit your config you must run `PackerCompile` make it take effect

```lua
  local saga = require('lspsaga')
  saga.init_lsp_saga({
    finder_action_keys = {
      open = '<CR>',
    },

  })

```

**Reply by haphamdev** [reply]
Thanks. `PackerCompile` works like a charm.
